### ISSUE TITLE
Globally disable gobal_search on admin class

### ISSUE THREAD

**Issue by webdevilopers** [sonata-project/SonataAdminBundle#3158]
According to this answer by @dianuj89
http://stackoverflow.com/questions/25909103/what-columns-get-searched-in-sonata-admin-global-serach
the following config should disable `global_search` for all my datagrid filters (though I think it is disabled by default?):

``` php
    protected function configureDatagridFilters(DatagridMapper $datagridMapper)
    {
        $datagridMapper
            ->add('id', null, array('global_search' => false))
            ->add('name', null, array('global_search' => false))
            ->add('branches', 'doctrine_orm_model_autocomplete', array(
                'global_search' => false
            ), null, array(
                'property' => 'name'
            ))
        ;
    }
```

When running the search I still get the admin class displayed though there are no results.
I expected it to disappear.

Do I have to override the template to achieve the behaviour?

**Comment by core23** [comment]
Feel free to contribute and change this behavior @webdevilopers

**Comment by webdevilopers** [comment]
Sorry but I'm no longer working on the projects I used Sonata with.

**Comment by jordisala1991** [comment]
This is fixed on #4941
